spread.yaml

```diff
@@ -336,14 +341,17 @@ suites:
             if [[ "$SPREAD_SYSTEM" != ubuntu-core-16-* ]]; then
                 . $TESTSLIB/pkgdb.sh
                 distro_purge_package snapd
-                distro_purge_package snap-confine
+                if [[ "$SPREAD_SYSTEM" != opensuse-* ]]; then
```

Comment:
It's always a tradeoff. Purging not existing packages is a very rare case where as installing packages is the very regular one. So limiting us to a minimal set of these is enough here.